### ISSUE TITLE
Survey 7 has no demographic information?

### ISSUE THREAD

**Issue by ianneidel** [kaijagahm/ygdpDashboard#42]
Trying to filter Survey 7 sentence participants by demographics has no effect. All participants are shown to have NA's for age, education, and race. As such applying changes in filtering (besides choosing to not include NA's which excludes all participants) does nothing.

Switching from a sentence from another survey which had filters applied to it also caused a server disconnect although I haven't yet been able to recreate it.

![Screen Shot 2021-01-04 at 4 54 20 PM](https://user-images.githubusercontent.com/68819605/103594667-34150180-4eae-11eb-9822-5efd4d64f3cf.png)

**Comment by kaijagahm** [comment]
Yeah, this is a problem with the data. I need to go back in the pipeline and figure out whether no demographic data was collected at all, or whether it got lost somewhere along the way. I think it's the former.

**Comment by kaijagahm** [comment]
There had been a problem in the original script that I used to process the survey 7 data. I've re-processed the survey 7 demographic information in the script "S7 Redo.Rmd". Added that to the database and then re-created snl.Rda using the "PTS.R" script in ygdpDashboard/. Seems to work now.

Associated commits:
https://github.com/kaijagahm/ygdpDashboard/commit/cf220e9b66b8a25785159195f453abaedc582341
